### DISCUSSION TITLE
Statidentifiers' value given by talents affects custom skills.

### DISCUSSION THREAD

**Discussion by Cyana-ir** [FakeFishGames/Barotrauma#14127] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

In recent update, there are some new talents like "start quest", "Junction Junkie" using AbilityConditionHasPermanentStat. But weirdly, It will affect custom skill bonus.
For example, "Cooking" skills of "Junction Junkie" engineer from Hugry Europans will puls 1 per fixes.
![}}{)%PS0M5YU2(JIBQ%MAEL](https://github.com/FakeFishGames/Barotrauma/assets/106088471/5baf1228-b603-43e2-ad11-079e0c0b9d7c)
![C@`~~E~P1AQ__WR6BTE@3LU](https://github.com/FakeFishGames/Barotrauma/assets/106088471/6210ab8a-7a8b-4162-a081-8ad32c674eb4)
In Fig.1. Cooking skill bonus are 3 after pick the talent (equal to 3 statidentifers' value as junctionjunkie_30&value=1 etc.)
In Fig.2. Cooking sikll bonus are 4 after repairing a electrical device ( I geuss its from <CharacterAbilityGivePermanentStat statidentifier="junctionjunkie_counter" value="1" removeondeath="false")****

### Reproduction steps

1. Start a game and enable any mods having the custom skill.
2. Using "setskill" to give any number of the custom skill to check the bonus.
3. Give exp or anyting elst to unlock a talent which using AbilityConditionHasPermanentStat to give statidentifier with value. Then check again.
4. Finish tasks given by the talent , the bonus should change by statidentifiers' value.( for start quest, bonus will decrease; for Junction Junkie, bonus will increase up to 100 , I guess)

Consol commands used as below(as for engineer talents affect Cooking skill):
enablecheats
setskill Cooking 10
giveexperience 450
spawnitem "EMP Grenade" inventory 1

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/ac2789e2fd82d811c61a87ca7ee240c6842f3837 (commit in our private repo). Coming up in a future update.
